### ISSUE TITLE
Annoying scrollbar

### ISSUE THREAD

**Issue by l1nlsp** [Xtrendence/X-Notify#1]
When the notification is invoked, it leaves an annoying scrollbar on the right side of the notification box, which doesn't look nice at all. I believe it can be solved by simply deleting the "overflow" property from the css in the "id=x-notify-container" element. 
![grafik](https://user-images.githubusercontent.com/51488017/122459532-a702c800-cf76-11eb-8aab-620223a4df61.png)
![grafik](https://user-images.githubusercontent.com/51488017/122459661-cb5ea480-cf76-11eb-84fa-25def0c9eb15.png)

**Comment by Xtrendence** [comment]
I'm guessing you're on Firefox. Sorry about that, I've added scrollbar-width:none; as the overflow is there in case there are enough notifications to fill the screen and require you to scroll through them. I've also set the overflow-y to auto so it doesn't show on any other browsers until actual scrolling is required. Forgot to test the notifications on Firefox so I missed that. Thank you very much for bringing it to my attention.
